bson/tests/test_objectid.py

```diff
@@ -511,18 +510,19 @@ def test_pid(self):
         self.assertTrue(oid_generated_on_client(ObjectId()))
 
     def test_generation_time(self):
-        d1 = datetime.datetime.utcnow()
+        d1 = datetime.datetime.now(datetime.UTC)
```

Comment:
Will it work in Py3.9?

Reply:
Darn -- nope.

Which is really annoying -- why depreciate when supported versions don't support the new way? 

Good reason to get the CI going! 

I don't have time to fix this right now, but I allowed maintainers to push to my branch, so you can do it -- or wait for me.

Reply:
Yep, and a PR for the CI/CD is already opened: https://github.com/py-bson/bson/pull/131

Reply:
OK -- fixed this